app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -119,6 +124,21 @@ public function store(Request $request, $id)
             $return_msg = trans('admin/users/message.declined');
         }
 
+        $item = $acceptance->checkoutable_type::find($acceptance->checkoutable_id);
```

Comment:
Let's get more info about the thing we're actually accepting. We should probably try to handle that via the model level, so we don't have to do it this janky way, but it mostly works this way. (It's just ugly and inelegant.)